components/brave_rewards/browser/promotion.h

```diff
@@ -21,6 +21,7 @@ struct Promotion {
   uint64_t expires_at;
   uint32_t type;
   uint32_t status;
+  bool legacyClaimed;
```

Comment:
no need to add this property here as client side doesn't need it

Reply:
Great - I wasn't sure so wanted to check, thanks!